MekHQ/data/scenariomodifiers/FacilityHostileDestroy.xml

```diff
@@ -2,6 +2,13 @@
 	<additionalBriefingText>Destroy or cripple all of the designated turrets.</additionalBriefingText>
 	<benefitsPlayer>false</benefitsPlayer>
 	<eventTiming>PreForceGeneration</eventTiming>
+	<linkedModifiers>
+		<entry>
+            		<key>FacilityHostileCapture.xml</key>
+            		<value>Capturing this facility will allow it to be destroyed as well.</value>
+		</entry>
+	</linkedModifiers>
+	<modifierName>Destroy</modifierName>
```

Comment:
Tab mixed with spaces? Seeing a lot of that in these XML files

Reply:
Ask the XML serializer.